server/Transaction.java

```diff
@@ -6,186 +6,168 @@
     String confirmation = "";
     double amount = 0.0;
     boolean valid = false;
-    Account fromAccount;
-    Account toAccount;
-    Customer toCustomer;
+    newbank.server.Account fromAccount;
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it